CMakeLists.txt

```diff
@@ -409,6 +359,10 @@ set(Boost_ADDITIONAL_VERSIONS
 # CMake'.  Good luck!  You'll need it.
 set(Boost_NO_BOOST_CMAKE 1)
 find_package(Boost "${BOOST_VERSION}")
+if(NOT Boost_FOUND)
+  get_deps(https://dl.bintray.com/boostorg/release/1.75.0/source/boost_1_75_0.tar.gz boost boost_1_75_0.tar.gz)
+  find_package(Boost)
```

Comment:
When we call `find_package()` the second time, perhaps we should mark it as `REQUIRED`?

Also, we need a little bit of special handling---some packagers need to know that the build will not use any HTTP or anything, so we have the `DISABLE_DOWNLOADS` CMake flag.  So, perhaps it is better to have some flow like this:

```
find_package(Boost "${BOOST_VERSION}")
if (NOT Boost_FOUND AND NOT DISABLE_DOWNLOADS)
  get_deps(...)
endif ()
find_package(Boost REQUIRED)
```

I'm not sure the syntax is exactly right, but you get the idea I think. :)